components/Layout/Layout.js

```diff
@@ -0,0 +1,14 @@
+import Nav from "../Footer/Nav";
+import Header from "../Header/Header";
+
+const Layout = ({ children }) => {
```

Comment:
I was a little confused about the term "Layout". Isn't this the main oder should it maybe be in the Home (index) component?

Reply:
its part of how to use the header an footer in the whole app in next.js. Let me know, if you want to know more, i can send you a link :)